### PR TITLE
ui: readability tweaks to logs page

### LINKED ISSUE #15874: ui: Densify the node logs page
There's a lot of whitespace on the logs page (`#/cluster/nodes/1/logs`) that makes the log-viewing experience less pleasant than it could be. People looking at logs in the UI are probably pretty accustomed to looking at raw text files, which have very little whitespace, so it shouldn't be too visually shocking to condense the page a bit.

For comparison, opening the raw log file in my browser shows more than 3x as many entries as our UI page. I wouldn't expect the UI to show quite as many, but that's a big difference:

screen shot 2017-05-11 at 12 29 41 pm

screen shot 2017-05-11 at 12 30 36 pm

@kuanluo for thoughts

### LINKED ISSUE #15990: ui: Newlines within log entries don't get rendered on logs page
Easy to reproduce by running with a high verbosity (e.g. `--verbosity=5`) and opening up the logs page (`/#/cluster/nodes/1/logs`):

screen shot 2017-05-17 at 12 22 46 pm

As first mentioned in https://github.com/cockroachdb/cockroach/issues/15958#issuecomment-302144025

### PR DESCRIPTION
**Pull request by vilterp**
Scratching a couple of itches:
- render newlines (fixes #15990)
- densify (fixes #15874)
- monospace font
- show loading spinner instead of "no data"

Before:
image

After:
image
(Scroll to right to see file & line)

### CODE REVIEW

**Comment by a-robinson** [comment]
Nice! What's the part of this change that makes renders newlines properly? `white-space pre`? I'd like to do something similar for the simulated allocator output on the range status debug page but hadn't previously known how.

**Comment by vilterp** [comment]
Yep, white-space pre makes the newlines actually newline.

**Review by BramGruneir — APPROVED** [review]

**Comment by BramGruneir** [comment]
Wonderful!  Another fix up I was going to tackle but didn't have time.  It's like you're reading my mind!

File: pkg/ui/src/views/cluster/containers/nodeLogs/index.tsx [hunk 1]
```diff
@@ -11,18 +11,21 @@ import { LongToMoment } from "src/util/convert";
 import { SortableTable } from "src/views/shared/components/sortabletable";
 import { AdminUIState } from "src/redux/state";
 import { refreshLogs, refreshNodes } from "src/redux/apiReducers";
-
 import { currentNode } from "src/views/cluster/containers/nodeOverview";
+import { CachedDataReducerState } from "oss/src/redux/cachedDataReducer";
```
Comment (couchand) [inline pkg/ui/src/views/cluster/containers/nodeLogs/index.tsx]: oss

**Pull request merged**